src/python/antares_xpansion/problem_generator_driver.py

```diff
@@ -118,7 +129,10 @@ def create_lp_dir(self):
 
     def lp_namer_options(self):
         is_relaxed = 'relaxed' if self.is_relaxed else 'integer'
-        ret = ["-a", str(self.output_path), "-f", is_relaxed]
+        if self.memory:
+            ret = ["--study", str(self.study_path), "-f", is_relaxed]  # study/output/xpansion_output_dir
+        else:
+            ret = ["-a", str(self.output_path), "-f", is_relaxed]
```

Comment:
and the third case e.g Mode FILE?

Reply:
Unsupported for e2e workflow.
I'll add an error case to explain it in code

Reply:
As per Remove_archive_and_file_support_in_launcher.md option mode is not supported in Launcher/E2E

I've added notes in ADRs that Archive mode is still supported for a time while study mode achieve feature completeness